libs/pages/elewa/about-us/src/lib/about-us-location-section/about-us-location-section.component.html

```diff
@@ -0,0 +1,19 @@
+<section class="home-about-us-location-section">
+    <iframe id="map" frameborder="0" style="border:0"
+      src="https://www.google.com/maps/embed/v1/place?key=AIzaSyCRWhaEwl7eyWlByjEdvDuPMn5CCIY2C9c
+      &q=3rd Floor, The Promenade, General Mathenge Rd, Nairobi" allowfullscreen>
```

Comment:
Do not push your API keys on public repos.

Reply:
removed